### ISSUE TITLE
Cannot update module in Foundry v11..."no module exists"

### ISSUE THREAD

**Issue by WildBillLives** [vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT#14]
### What happened?

I get the following message trying to update the OSE Advanced Fantasy (premium) module in Foundry v11: 

Failed to install module: Unable to load valid module manifest data from "https://github.com/vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT/releases/download/latest/module.json" The requested manifest at "https://github.com/vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT/releases/download/latest/module.json" did not provide module manifest data. 

### What is the expected behaviour?

It should update the module...

### Relevant Errors and Warnings

```shell
"Failed to install module: Unable to load valid module manifest data from "https://github.com/vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT/releases/download/latest/module.json" The requested manifest at "https://github.com/vttred/Old-School-Essentials-Advanced-Fantasy-for-Foundry-VTT/releases/download/latest/module.json" did not provide module manifest data."
```

### Additional Support Details

Foundry v11 stable build 299

### Foundry VTT Core Version

V11

### Old-School Essentials Core Version

v1.9.1

### Please update your browser regularly. Which (updated) browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by anthonyronda** [comment]
Thanks for making the report. This is already fixed and awaiting release in 1.0.2 along with the other fixes

**Comment by anthonyronda** [comment]
@WildBillLives the fix is available in the latest release, but you will still have to uninstall and reinstall the module in order to get it. After that, you may use the update button normally.
